rs/rosetta-api/Cargo.toml

```diff
@@ -7,8 +7,8 @@ edition = "2021"
 default-run = "ic-rosetta-api"
 
 [dependencies]
-actix-rt = "2.10.0"
-actix-web = "4.9.0"
+actix-rt = { workspace = true }
+actix-web = { workspace = true }
```

Comment:
It should be possible to move the `rolling-file = "0.2.0"` dependency below to the workspace also? Elsewhere it's only used in `rs/rosetta-api/icrc1/rosetta/Cargo.toml` (and it's the same version there).

Reply:
Yes I can do that.

Reply:
Done